docs/klaytn-contracts/dao-contracts/sdks-and-tutorials.md

```diff
@@ -3,3 +3,567 @@ title: Tutorial
 sidebar_label: Tutorial
 ---
 
+# 💻 Simple On-chain Governance Tutorial
+
+In this tutorial you will use [Klaytn Contract](https://github.com/klaytn/klaytn-contracts), [Klaytn Contract Wizard](https://wizard.klaytn.foundation/)  and [Caver-Js](https://docs.klaytn.foundation/content/dapp/sdk/caver-js/getting-started) to run a simple on-chain governance. You tend to learn about how to:
+* Create a KIP7 governance token contract that you ll use to cast votes on the governance proposals.
+* Create a standard governor contract.
+* Create a mintable KIP7 contract that will mint new tokens on the successful execution of the on-chain governance proposal.
+
+## 1. Prerequisite <a id="KIP7 Tutorial Prerequsite"></a>
+
+* [Metamask](https://metamask.io/download/): Used to deploy the contracts interact with the contracts.
+* [Remix IDE](https://remix.ethereum.org/): Used to compile the contracts and deploy through MetaMask.
+* [Klaytn Plugin on Remix](https://klaytn.foundation/using-klaytn-plugin-on-remix/): develop, deploy, and interact with smart contracts on the Klaytn blockchain and execute various Klaytn-specific transactions
+* Test KLAY from [Faucet](https://baobab.wallet.klaytn.foundation/faucet): Fund your account with sufficient KLAY
+* [Node Js and NPM](https://kinsta.com/blog/how-to-install-node-js/)
+
+## 2. Create and deploy your KIP7 governance token <a id="Create and deploy your KIP7 governance token"></a>
+
+In this section, you'll create your simple governance token—the token you will use to vote on proposals in the governance contract.
+
+**Klaytn Contract Wizard**
+
+* Open [Klaytn Contract Wizard](https://wizard.klaytn.foundation/)
+* Select **KIP7**.
+* Provide a `name` and a `symbol` for the token. For example: *GovernanceToken*, *GTK*.
+* In **Premint**, provide the number of tokens to issue to your account. For example: *100*.
+* In **Feature**, select **Votes**.
+* Click **Open** in **Remix IDE**.
+
+![](/images/dao/daoGovernanceToken.png)
+
+**Remix IDE**
+* In Remix, click **Compile contract**.
+* Click the **Klaytn tab** on your left having installed the plugin
+* Select **Environment** > **Injected Provider** - **MetaMask**.
+* In **Contract**, select your contract. For example, *GovernanceToken*.
+* Click **Deploy**.
+
+This will engage your MetaMask to deploy the contract to the Klaytn Baobab Testnet through your currently selected MetaMask account. Click **Confirm** in the MetaMask modal.
+
+![](/images/dao/daoRemixGovernanceToken.png)
```

Comment:
Numbering in the image and the points does not match. either we change the numbers in the image or give numbers to the above bullet points